server/server_test.go

```diff
@@ -787,31 +791,105 @@ func TestCrossClientScopes(t *testing.T) {
 	if err := s.storage.CreateClient(peer); err != nil {
 		t.Fatalf("failed to create client: %v", err)
 	}
+	fixture.peer = &peer
 
-	oauth2Config = &oauth2.Config{
-		ClientID:     client.ID,
-		ClientSecret: client.Secret,
-		Endpoint:     p.Endpoint(),
-		Scopes: []string{
-			oidc.ScopeOpenID, "profile", "email",
-			"audience:server:client_id:" + client.ID,
```

Comment:
Why don't we just remove this claim for the test?

Reply:
I wanted to ensure backward-compatibility so I left an old test (if clientID is explicitly requested, issued ID token should still contain only one occurrence of clientID in audience claim)